packages/react-scripts/template/src/components/example/ArtifactsViewer.js

```diff
@@ -100,62 +115,11 @@ const PhotoViewer = ({ photos, height = 250 }) => {
 
   return (
     <div css={photoViewerCss} style={{ height }}>
-      {photos.map((photo) => (
+      {photos.photos.map((photo) => (
```

Comment:
should this be `data.photos`, or can we destructure somewhere to avoid the double `photos.photos`? That always feels weird to me.